scripts/get-latest-rhocp-repo.sh

```diff
@@ -20,20 +25,25 @@ if ! sudo subscription-manager status >&/dev/null; then
     exit 1
 fi
 
-# Get minor version of currently checked out branch.
-# It's based on values stored in Makefile.version.$ARCH.var.
-make_version="${REPOROOT}/Makefile.version.$(uname -m).var"
-if [ ! -f "${make_version}" ] ; then
-    # Attempt to locate the Makefile version file next to the current script.
-    # This is necessary when bootstrapping the development environment for the first time.
-    make_version=$(find "${SCRIPTDIR}" -maxdepth 1 -name "Makefile.version.$(uname -m).*.var" | tail -1)
+if [[ "$#" -eq 1 ]]; then
+    current_minor="${1}"
+    stop="${current_minor}"
+else
+    # Get minor version of currently checked out branch.
+    # It's based on values stored in Makefile.version.$ARCH.var.
+    make_version="${REPOROOT}/Makefile.version.$(uname -m).var"
```

Comment:
Does this work when bootstrapping the devenv from scratch?

Reply:
Well, this part is not changed, only leading spaces were added